internal/service/ec2/ec2_ami_data_source.go

```diff
@@ -217,6 +217,10 @@ func dataSourceAMI() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"uefi_data": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
The same in `internal/service/ec2/ec2_ami_from_instance.go` also as the `aws_ami_from_instance` resource's schema must match.
Thanks 👏.

Reply:
Done :)